src/utils/fetch.test.js

```diff
@@ -0,0 +1,64 @@
+import { expect } from 'chai';
+import nock from 'nock';
+
+import fetch, { HTTPResponseError, DEFAULT_TIMEOUT } from './fetch.js';
+
+describe('Fetch', () => {
+  describe('Response parsing', () => {
+    it('parses response as JSON', async () => {
+      const scope = nock('https://example.com').get('/').reply(200, { data: 'success' });
```

Comment:
I am not sure I understand the name “scope” in this context 🤔

Reply:
I understand, it's totally clear to me, I just followed [the nock documentation](https://github.com/nock/nock) way.